README.md

```diff
@@ -31,6 +31,20 @@ except: ['camelize'] // imports all helpers except `camelize`
 only: ['camelize'] // imports only `camelize`
 ```
 
+### Addons
+
+Similar to the configuration above, but place it in the `options` hash of your `index.js` file. If a consuming application also uses `ember-cli-string-helpers` as well, the app and addon configuration will be merged.
```

Comment:
> the app and addon configuration will be merged

This is what I figured out with my own experimentation. If there are any caveats to this (or if it's flat out wrong) let me know.